Src/DependencyCollector/Shared/Implementation/ClientServerDependencyTracker.cs

```diff
@@ -10,7 +10,7 @@
 
     internal static class ClientServerDependencyTracker
     {
-        internal const string DependencyActivityName = "Microsoft.ApplicationInsights.Web.Dependency";
+        internal const string DependencyActivityName = "Microsoft.ApplivationInsights.Web.Dependency";
```

Comment:
Is there any code that already took a dependency on the previous Activity Name? (E.g. consumes activity and validates it against the misspelled name?) I assume not by looking at an `internal` in the definition.